tests/nexus_api_test.go

```diff
@@ -511,23 +570,6 @@ func (s *ClientFunctionalSuite) TestNexusCancelOperation_Outcomes() {
 				s.Equal("deliberate internal failure", unexpectedError.Failure.Message)
 			},
 		},
-		// TODO: This can't be tested without the test taking over a minute since this is the default matching
```

Comment:
Wondering why you didn't test this here.

Reply:
Would have had to refactor the test cases for them to all set a header field which would have only been different for this test case. Seemed like a separate test method was easier